### PR TITLE
Feat : add  Modal and navbar

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Nice work! Used clean code practices

File: Components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.search-bar input {
+    border: none;
+    outline: none;
+    font-family: inherit;
+    padding: 0.5rem;
+    background-color: transparent;
+    width: 25vw;
+    max-width: 600px !important;
```
Comment (shivampandey0) [inline Components/navbar.css]: Vikram: "using `!important` says there is something wrong "

fix `!important` by specificity or source order

**Review by shivampandey0 — COMMENTED** [review]
Looks good.
Added a suggestion

File: Components/navbar.css (hunk 1 above)
Comment (ZuberDunge) [reply Components/navbar.css]: Thanks for noticing. Fixed.

Comment (shivampandey0) [reply Components/navbar.css]: Good to merge

**Pull request merged**